src/utils/rfq_client.ts

```diff
@@ -0,0 +1,107 @@
+import {
+    IRfqClient,
+    RfqClientV1PriceRequest,
+    RfqClientV1PriceResponse,
+    RfqClientV1QuoteRequest,
+    RfqClientV1QuoteResponse,
+} from '@0x/asset-swapper';
+import { AxiosInstance } from 'axios';
+import { OK } from 'http-status-codes';
+
+import { RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { logger } from '../logger';
+
+export class RfqClient implements IRfqClient {
+    constructor(private readonly _rfqApiUrl: string, private readonly _axiosInstance: AxiosInstance) {}
+
+    /**
+     * Communicates to an RFQ Client to fetch available prices
+     */
+    public async getV1PricesAsync(request: RfqClientV1PriceRequest): Promise<RfqClientV1PriceResponse> {
+        try {
+            const response = await this._axiosInstance.post(
+                `${this._rfqApiUrl}/rfqt/v1/prices`,
+                {
+                    altRfqAssetOfferings: request.altRfqAssetOfferings,
```

Comment:
(for my understanding) what's the reason for not directly using `request`?

Reply:
No real reason - I think I was playing around with some of the parameters and wanted to selectively send only some of them. Might change it to just sending the `request` object though